src/ompl/tools/thunder/SPARSdb.h

```diff
@@ -846,6 +856,9 @@ namespace ompl
 
             double heuristicScaling_ {1.0};
 
+            /** \brief optional variable. If set, will sample each segment in the path with this many nodes and attempt to insert */
```

Comment:
extra \?

Reply:
This is usually how they denote brief in ompl I think